### DISCUSSION TITLE
支付宝小程序使用，如何根据页面的数据变化，重新渲染折线图和饼图

### DISCUSSION THREAD

**Discussion by gaoxuzhao** [antvis/G2#5885] [Q&A]
- [ ] I have searched the [issues](https://github.com/antvis/g2/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
解决页面输入条件时，图片无法变化

### What does the proposed API look like?
有个重新渲染图表的api

**Comment by pearmini** [comment]
文档里面有介绍：https://g2.antv.antgroup.com/manual/core/data

**Reply by gaoxuzhao** [reply]
请问，有没有专门针对 支付宝小程序的使用说明呢？

**Reply by pearmini** [reply]
这个没有
